.circleci/comment.js

```diff
@@ -40,6 +40,19 @@ async function run() {
             break;
           }
         }
+      } else {
+        //If it isn't a PR commit, then we are on main. Create a comment for the test app and docs build
+        await octokit.repos.createCommitComment({
```

Comment:
Is this why I've been getting lots of random notifications on commits? Is this going to be annoying?

I think it might not only happen on main also, because you can commit to branches that aren't in PRs.

Reply:
Ah, lemme go ahead and add a more strict check for main. I'm surprised you've been getting notifications on your commits, shouldn't this change be isolated to this branch?

The ideal scenario is that this only runs on commits in main, open to suggestions on the desired cadence. I think it would be nice if it ran only on the commit before publish/release but wasn't able to think up of a way to do so

Reply:
No, not on my commits. I think maybe they were the commits on this PR. But they link to a commit rather than a PR.

Reply:
Ah right, it was spammy because I was doing some frequent commits to handle issues between ci vs local differences, apologies about that. When we merge and have it run on commits to main only it shouldn't be as spammy (at least no less spammy than a standard PR with its comment per commit push)